### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by LimerBoy** [ragardner/tksheet#7]
Hi, how can I get information from a table?

**Comment by ragardner** [comment]
What sort of information?

All available functions are inside the `_tksheet.py` file but you can use these functions to get cell data

```python
my_sheet.get_cell_data(r, c, return_copy = True)

my_sheet.get_row_data(r, return_copy = True)

my_sheet.get_column_data(c, return_copy = True)
```
`return_copy` is whether or not you want to get your sheets actual variables or get copies of them

**Comment by LimerBoy** [comment]
![py](https://user-images.githubusercontent.com/43149460/70982704-1ec10f80-20c0-11ea-8a3c-ebe984deaeaf.PNG)

get_cell_data Return None

**Comment by ragardner** [comment]
Sorry yes it was a typo in the function leading it to return from another function instead of the table, it should be fixed in 4.1.7 released just now

**Comment by LimerBoy** [comment]
Well, I already fixed this in my code. It turned out like this:

![зображення](https://user-images.githubusercontent.com/43149460/70983817-0ce06c00-20c2-11ea-9403-892033f27095.png)
